requirements/dev_python27.txt

```diff
@@ -1,11 +1,2 @@
--r base-py2.txt
-
-mock>=2.0.0
-apache-libcloud>=0.14.0
-boto>=2.32.1
-boto3>=1.2.1
-moto>=0.3.6
-SaltPyLint>=v2017.3.6
-pytest>=3.5.0
-git+https://github.com/eisensheng/pytest-catchlog.git@develop#egg=Pytest-catchlog
-git+https://github.com/saltstack/pytest-salt.git@master#egg=pytest-salt
+# This is for legacy purposes
```

Comment:
I know this is picky, but it might make sense for this comment to match the one in `dev_python34.txt` (and I liked that one the best of the two). 😄

Reply:
Will do